backend/cms/models/event.py

```diff
@@ -170,12 +170,16 @@ class EventTranslation(models.Model):
     """
     Database object representing an event tranlsation
     """
+    DRAFT = 'DRAFT'
+    REVIEW_PENDING = 'PENDING'
+    REVIEW_FINISHED = 'FINISHED'
+
     STATUS = (
-        ('draft', 'Entwurf'),
```

Comment:
Using constants here is more flexible and  less error-prone than inline strings

Reply:
agree on that.